jwst/resample/resample.py

```diff
@@ -377,6 +377,14 @@ def drizzle_arrays(insci, inwht, input_wcs, output_wcs, outsci, outwht, outcon,
         if outcon.ndim == 3:
             outcon = outcon[planeid]
 
+        if xmin is xmax is ymin is ymax is None:
```

Comment:
I did not know this kind of syntax worked!  Cool.  I assume it treats the multiple arguments with a logical OR, i.e. if any one of them is None, it's True? Or is it an AND?

Reply:
Not a very intuitive syntax, IMO but it's OK. I am actually not sure why it is necessary to check all of them. Wouldn't  just `xmin is None` be enough? What will happen if `xmin` is `None` and `xmax=100` (for example)?

Reply:
Yeah, checking one is probably fine.  I'm the only one calling all this internal code anyway.

This only works for `None` as it is a singleton, i.e. there is only of it.  There's no instance or copy.  So if `xmin is None` and `xmax is None`, they are both just the same thing, so `xmin is xmax`.